debian/libauxv1.install

```diff
@@ -0,0 +1,5 @@
+usr/bin/lsauxv
+usr/share/man/man1/lsauxv.1
+usr/share/man/man3/auxv.3
```

Comment:
Our automake rules create an extra link (libauxv.3 -> auxv.3). Do you want this in debian?

PS: I thought it would be fair to mention it, because I am not very familiar with the debian packaging guidelines.